packages/dscc-gen/src/files.ts

```diff
@@ -27,19 +25,35 @@ const ENCODING = 'utf8';
 const CURR_DIR = process.cwd();
 
 const fixFile = (templates: Template[]) => async (file: string) => {
-  const contents = await util.readFile(file, ENCODING);
+  const contents = util.readFile(file, ENCODING);
   const newContents = templates.reduce(
     (acc, {match, replace}) => acc.replace(match, replace),
     contents
   );
   return util.writeFile(file, newContents, ENCODING);
 };
 
+export const recursiveReaddirSync = async (
+  baseDirectory: string
+): Promise<string[]> => {
+  const dirContents = fs.readdirSync(baseDirectory, {withFileTypes: true});
+  const files: string[][] = await Promise.all(
+    dirContents.map(async (item) => {
+      const itemPath: string = path.resolve(baseDirectory, item.name);
+      return item.isDirectory()
+        ? await recursiveReaddirSync(itemPath)
+        : [itemPath];
+    })
+  );
+  return files.reduce((acc, fileNames) => {
+    return acc.concat(fileNames);
+  }, []);
+};
 export const fixTemplates = async (
   baseDirectory: string,
   templates: Template[]
 ): Promise<boolean> => {
-  const filesToUpdate = await listFiles(baseDirectory, ['node_modules']);
+  const filesToUpdate = await recursiveReaddirSync(baseDirectory);
```

Comment:
The new function should still be called list files & we need to make sure it doesn't include node_modules.

Reply:
done